inference-engine/ie_bridges/c/samples/hello_classification/README.md

```diff
@@ -47,33 +47,45 @@ To run the sample, you need specify a model and image:
 >
 > - The sample accepts models in ONNX format (\*.onnx) that do not require preprocessing.
 
-You can do inference of an image using a trained AlexNet network on a GPU using the following command:
+### Example
+1. Download a pre-trained model using [Model Downloader](@ref omz_tools_downloader_README):
+```
+python <path_to_omz_tools>/downloader.py --name <model_name>
+```
+
+2. If a model is not IR or ONNX format, it must be converted. You can do this using the special script:
 
-```sh
-./hello_classification_c <path_to_model>/alexnet_fp32.xml <path_to_image>/cat.png GPU
+```
+python <path_to_omz_tools>/converter.py --name <model_name>
+```
+
+3. Perform inference of `car.bmp` using `alexnet` model on a `GPU`, for example:
+
+```
+<path_to_sample>/hello_classification_c <path_to_model>/alexnet.xml <path_to_image>/car.bmp GPU
 ```
 
 ## Sample Output
 
 The application outputs top-10 inference results.
 
-```sh
+```
 Top 10 results:
 
-Image /opt/intel/openvino/deployment_tools/demo/car.png
+Image C:\images\car.bmp
```

Comment:
use path from demo folder

Reply:
It's about the images that can be downloaded from https://storage.openvinotoolkit.org/data/test_data